Source/JavaScriptCore/wasm/WasmSlowPaths.cpp

```diff
@@ -415,6 +415,22 @@ WASM_SLOW_PATH_DECL(array_new)
     WASM_RETURN(Wasm::arrayNew(instance, instruction.m_typeIndex, size, value));
 }
 
+WASM_SLOW_PATH_DECL(array_new_data)
+{
+    auto instruction = pc->as<WasmArrayNewData>();
+    uint32_t arraySize = READ(instruction.m_size).unboxedUInt32();
+    uint32_t offset = READ(instruction.m_offset).unboxedUInt32();
+
+    auto result = Wasm::operationWasmArrayNewData(instance,
```

Comment:
This is not OK. We should not call operation function from C++. You need to extract code into WasmOperationsInline.h